{"uid":"b0e9a860c30edb18","name":"Scenario Outline name: Verify that the Update API will be fails if recorded has future or invalid formatted date","fullName":"features.APITests.steps.test_update_steps#test_scenario_outline_name_verify_that_the_update_api_will_be_fails_if_recorded_has_future_or_invalid_formatted_date","historyId":"f85dfc7bbd6316b3378cd2d4ecf31ab0","time":{"start":1760009772399,"stop":1760009776816,"duration":4417},"description":"/home/runner/work/imms_fhir_api_automation/imms_fhir_api_automation/features/APITests/update.feature: Scenario Outline name: Verify that the Update API will be fails if recorded has future or invalid formatted date","descriptionHtml":"<p>/home/runner/work/imms_fhir_api_automation/imms_fhir_api_automation/features/APITests/update.feature: Scenario Outline name: Verify that the Update API will be fails if recorded has future or invalid formatted date</p>\n","status":"failed","statusMessage":"AssertionError: FAILURE: check bool(False): Expected Diagnostics: Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone- Date must not be in the future., got Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone-  Date must not be in the future.\nfeatures/APITests/steps/common_steps.py:205 in validateForbiddenAccess() -> validate_error_response(error_response, ErrorName[errorName].value, context.ImmsID)\nutilities/api_fhir_immunization_helper.py:104 in validate_error_response() -> check.is_true(\n\n------------------------------------------------------------\nFailed Checks: 1","statusTrace":"FAILURE: check bool(False): Expected Diagnostics: Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone- Date must not be in the future., got Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone-  Date must not be in the future.\nfeatures/APITests/steps/common_steps.py:205 in validateForbiddenAccess() -> validate_error_response(error_response, ErrorName[errorName].value, context.ImmsID)\nutilities/api_fhir_immunization_helper.py:104 in validate_error_response() -> check.is_true(\n\n------------------------------------------------------------\nFailed Checks: 1","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setup_environment","time":{"start":1760009223884,"stop":1760009223884,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"global_context","time":{"start":1760009223886,"stop":1760009225414,"duration":1528},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"context","time":{"start":1760009772399,"stop":1760009774145,"duration":1746},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"_pytest_bdd_example","time":{"start":1760009817052,"stop":1760009817052,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"description":"/home/runner/work/imms_fhir_api_automation/imms_fhir_api_automation/features/APITests/update.feature: Scenario Outline name: Verify that the Update API will be fails if recorded has future or invalid formatted date","status":"failed","statusMessage":"AssertionError: FAILURE: check bool(False): Expected Diagnostics: Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone- Date must not be in the future., got Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone-  Date must not be in the future.\nfeatures/APITests/steps/common_steps.py:205 in validateForbiddenAccess() -> validate_error_response(error_response, ErrorName[errorName].value, context.ImmsID)\nutilities/api_fhir_immunization_helper.py:104 in validate_error_response() -> check.is_true(\n\n------------------------------------------------------------\nFailed Checks: 1","statusTrace":"FAILURE: check bool(False): Expected Diagnostics: Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone- Date must not be in the future., got Validation errors: recorded must be a valid datetime in one of the following formats:- 'YYYY-MM-DD' — Full date only- 'YYYY-MM-DDThh:mm:ss%z' — Full date and time with timezone (e.g. +00:00 or +01:00)- 'YYYY-MM-DDThh:mm:ss.f%z' — Full date and time with milliseconds and timezone-  Date must not be in the future.\nfeatures/APITests/steps/common_steps.py:205 in validateForbiddenAccess() -> validate_error_response(error_response, ErrorName[errorName].value, context.ImmsID)\nutilities/api_fhir_immunization_helper.py:104 in validate_error_response() -> check.is_true(\n\n------------------------------------------------------------\nFailed Checks: 1","steps":[],"attachments":[{"uid":"1d5146c375ccdd13","name":"Step Passed: I have created a valid vaccination record","source":"1d5146c375ccdd13.txt","type":"text/plain","size":60},{"uid":"3999fb13393089d7","name":"Step Passed: Send a update for Immunization event created with recorded being updated to 'future_date'","source":"3999fb13393089d7.txt","type":"text/plain","size":108},{"uid":"4fef3cfcb1ab4c32","name":"Step Passed: The request will be unsuccessful with the status code '400'","source":"4fef3cfcb1ab4c32.txt","type":"text/plain","size":78},{"uid":"f9151207883d28e2","name":"Step Passed: The Response JSONs should contain correct error message for 'invalid_recorded'","source":"f9151207883d28e2.txt","type":"text/plain","size":97},{"uid":"4edcf94aacc8a1b1","name":"stdout","source":"4edcf94aacc8a1b1.txt","type":"text/plain","size":8890}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":5,"hasContent":true},"afterStages":[],"labels":[{"name":"epic","value":"Immunization Service"},{"name":"suite","value":"Update the immunization of a patient"},{"name":"feature","value":"Update the immunization of a patient"},{"name":"tag","value":"Update_Feature"},{"name":"tag","value":"vaccine_type_RSV"},{"name":"tag","value":"patient_id_Mod11_NHS"},{"name":"tag","value":"Delete_cleanUp"},{"name":"tag","value":"supplier_name_Postman_Auth"},{"name":"parentSuite","value":"features.APITests.steps"},{"name":"host","value":"runnervmwhb2z"},{"name":"thread","value":"2491-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"features.APITests.steps.test_update_steps"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"_pytest_bdd_example","value":"{'Date': 'future_date'}"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":["Update_Feature","supplier_name_Postman_Auth","vaccine_type_RSV","patient_id_Mod11_NHS","Delete_cleanUp"]},"source":"b0e9a860c30edb18.json","parameterValues":["{'Date': 'future_date'}"]}